src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -762,6 +825,26 @@ private function checkoutUsing(
         $this->proceedSelectingShippingAndPaymentMethod($order, $shippingMethod, $paymentMethod);
     }
 
+    private function createShippingPaymentMethodsAndAddress(): void
+    {
+        /** @var AddressInterface $address */
+        $address = $this->addressFactory->createNew();
+        $address->setCity('Wawa');
+        $address->setCountryCode($this->countryRepository->findOneBy([])->getCode());
+        $address->setFirstName('Jon');
+        $address->setLastName('Doe');
+        $address->setPostcode('000');
+        $address->setStreet('Happy');
+
+        $this->theCustomerAddressedItToWithIdenticalBillingAddress($address);
+
+        $shippingMethod[] = $this->shippingMethodRepository->findOneBy([]);
+
+        $paymentMethod[] = $this->paymentMethodRepository->findOneBy([]);
```

Comment:
```suggestion
        $paymentMethod = $this->paymentMethodRepository->findOneBy([]);
```
and check if it is not null